src/foam/lang/Enum.js

```diff
@@ -442,7 +442,11 @@ foam.CLASS({
     {
       name: 'toJSON',
       value: function(value) { return value.ordinal; }
-    }
+    },
+    [
+      'isDefaultValue',
+      function(v) { return false }
```

Comment:
```suggestion
      function(v) { return v === undefined || v === null }
```
This should still return true if the value is empty